pdns/dnsdistdist/docs/changelog.rst

```diff
@@ -37,6 +425,12 @@ Changelog
 
     Only add EDNS on negative answers if the query had EDNS
 
+  .. change::
+    :tags: Improvements
+    :pullreq: 10035
```

Comment:
is this 1.5.1 change in the 1.6.0 PR on purpose?

Reply:
How the h*ll did I manage to do that? Fixing, thanks!